app/controllers/books_controller.rb

```diff
@@ -17,18 +17,14 @@ def new
 
   def create
     @book = Book.new(book_params)
-    raise ActiveRecord::RecordInvalid, @book unless @book.valid?
-    raise ActiveRecord::RecordNotSaved, @book unless @book.save
-
-    flash[:success] = 'Book create completed'
+    @book.save!
```

Comment:
this one please consider when error happen to frontend too. Frontend will only get status 500 due to we not catch any error and response it properly